portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,51 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Raul's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="topbar">
+      <ul>
+        <li class="topbartext">Raul Cruz</li>
+        <li class="logo1">
+          <a href="https://github.com/raulcruise" target="_blank">
+            <img src="images/github.png" alt="GitHub logo">
+          </a>
+        </li>
+        <li class="logo2">
+          <a href="https://www.linkedin.com/in/raulcruise/" target="_blank">
+            <img src="images/linkedin.png" alt="LinkedIn logo">
+          </a>
+        </li>
+      </ul>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 class="abouttitle">About Me</h1>
```

Comment:
I see that you used concatenatedstring pattern. On the style guide I noticed that they tend to use the dash-separated-string pattern. But this is not a required suggestion. In the future, you may want to use the dash-separated-string pattern in HTML.

Reference:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks

Reply:
Done.